empiric-package/empiric/core/types.py

```diff
@@ -0,0 +1,12 @@
+from typing_extensions import Literal
+
+ADDRESS = int
+HEX_STR = str
+
+# Network Types
+STAGING = "staging"
+TESTNET = "testnet"
+INTEGRATION = "integration"
+MAINNET = "mainnet"
```

Comment:
Is there value to importing these from starknet.py? If I understand correctly given that we pass these into Network configs from starknet.py we should use their constants, else could break

Reply:
I believe the only issue was their library doesn’t directly support integration network

Reply:
Starknet.py defines `network` as Union[Literal['mainnet', 'testnet'].

Reply:
yeah but our Literal is a superset of theres.  If you import their MAINNET constant, it's the same as ours.

https://github.com/software-mansion/starknet.py/blob/development/starknet_py/net/networks.py#L9

if you check here it just has:

```
MAINNET = "mainnet"
TESTNET = "testnet"
```

So we already do support their network constants